### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,19 +1,60 @@
...
+basepython = python3
+skip_install = true
+deps =
+    flake8
+    check-manifest
+    check-python-versions
+commands =
+    flake8 src setup.py
+    check-manifest
+    check-python-versions .
```
Comment (mgedmin) [inline tox.ini:33]: FWIW `.` is the default location for `check-python-versions` and doesn't need to be explicitly specified.

File: tox.ini [hunk 2]
```diff
@@ -1,19 +1,60 @@
...
+plugins = coverage_python_version
+source = src
+
+[coverage:report]
+precision = 2
+exclude_lines =
+    pragma: nocover
+    except ImportError:
+    raise NotImplementedError
+    if __name__ == '__main__':
```
Comment (mgedmin) [inline tox.ini:57]: FWIW the old .coveragerc also had the following in exclude_lines:

```
    self.fail
    raise AssertionError
```

We might want to look into that, if we notice a coverage drop.

(The GitHub review UI is bad in some respects: I can't see commit statuses while I'm writing review comments, unless I open new tabs and poke around.)

Comment (mgedmin) [reply tox.ini:57]: Coveralls did not leave a commit status, so I guess we're not tracking coverage for this package.

(Or coveralls once again ran into GitHub API rate limits and can't post commit statuses.)

File: tox.ini (hunk 1 above)
Comment (icemac) [reply tox.ini:33]: Thank you for the info I am going to remove it.

File: tox.ini (hunk 2 above)
Comment (icemac) [reply tox.ini:57]: I can't even log in to coveralls.io due to a GitHub-Error in the log-in process.

So I am going to add the missing ignores blindly.

**Pull request merged**